README.md

```diff
@@ -33,32 +44,6 @@ const config = {
 };
 ```
 
-<details><summary markdown="span"><strong>For LoopBack 3 users</strong></summary>
```

Comment:
Since `6.x` will just for LB4, do we want to remove all the collapsible sections in REAME?

Reply:
> Since `6.x` will just for LB4, do we want to remove all the collapsible sections in README?

Yes please :+1: